testsuite/oauth/authorize.py

```diff
@@ -99,7 +99,12 @@ def _browser(self):
         """
         RemoteConnection.set_timeout(CONNECTION_TIMEOUT)
 
-        return webdriver.Firefox()
+        profile = webdriver.FirefoxProfile()
+        profile.set_preference('general.useragent.override', 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.116 Safari/537.36')  # noqa
```

Comment:
Can you add a comment explaining this? And does this override work with all the other vendors? Should we extarct this out so it can be configured on a per-vendor basis?